test/config-serializer.spec.js

```diff
@@ -29,9 +50,40 @@ describe('Config Serializer', () => {
     };
 
     let str = serializeConfig(cfg);
-    expect(str).to.be.equal(outCfg);
+    expect(str).toBe(outCfg);
+  });
+
+  it('can serialize System configuration', () => {
```

Comment:
If I understand the purpose correctly, I'd rename the test to 
'can serialize configuration to System'
or 
'can serialize configuration to System format'